gensim/test/test_notebooks.py

```diff
@@ -0,0 +1,82 @@
+import sys
+import tempfile
+from glob import glob
+
+import nbformat
+from nbconvert.preprocessors import ExecutePreprocessor
+from nbconvert.preprocessors.execute import CellExecutionError
+
+import logging
+import unittest
+import os
+import os.path
+import tempfile
+import numbers
+import codecs   
+
+class TimeoutError(Exception):
```

Comment:
Removing it gives me this error on timeout: 
```NameError: global name 'TimeoutError' is not defined```

Is this a python2 vs python3 issue? I understand that TimeoutError is a subclass of OSError but I'm not sure how to use it in code.